tests/conftest.py

```diff
@@ -155,3 +156,14 @@ def lang_data_many(lang_type, lic_type, lang_data, service, identity):
         service.create(identity, data)
     Vocabulary.index.refresh()  # Refresh the index
     return lang_ids
+
+
+# FIXME: https://github.com/inveniosoftware/pytest-invenio/issues/30
+# NOTE: Without this, success of test depends on test order
```

Comment:
I learnt this maybe a few weeks ago... It turns out that `NOTE` has a specific meaning for translations (i.e. this will be picked up by babel)... So for notes we should just put the comment :) If you see any around the code, it was probably me... I'm trying to remove them when I see them 😅 

```suggestion
# without this, the success of test depends on the tests order
```

Reply:
Nice!